test/verifiedTests/analysis/testMTA/testMTA.m

```diff
@@ -0,0 +1,84 @@
+% The COBRAToolbox: testMTA.m
+%
+% Purpose:
+%     - Check the functions MTA and rMTA using different solvers
+%
+% Authors:
+%     - Luis V. Valcarcel 2018-12-17
+%
+
+global CBTDIR
+% prepareTest('needsMIQP', true, 'useSolversIfAvailable', {'tomlab_cplex', 'ibm_cplex', 'gurobi'});
+prepareTest('needsQP', true, 'useSolversIfAvailable', {'tomlab_cplex', 'ibm_cplex', 'gurobi'});
```

Comment:
this should be `solverPkgs = prepareTest('needsQP', true, 'useSolversIfAvailable', {'tomlab_cplex', 'ibm_cplex', 'gurobi'});`
and later use the respective solverPackage.QP /LP etc fields.